manifests/pipelines/aiedge-e2e.pipeline.yaml

```
@@ -309,10 +318,8 @@ spec:
       name: openshift-client
   - name: test-model-rest-svc
     params:
-    - name: model-name
-      value: $(params.model-name)
-    - name: model-version
-      value: $(params.model-version)
+    - name: deployment-name
```

I don't want to be nitpicking, it's just this is in fact a name of the service. Even the task is named `test-model-rest-svc`, so maybe we should align it?
```suggestion
    - name: service-name
```